keyserver/README.md

```diff
@@ -36,7 +36,9 @@ Use "coniksserver [command] --help" for more information about a command.
 is for the registration proxy, the second is the server's public address
 for "read-only" requests (lookups, monitoring etc).
 - Edit the configuration file as needed:
-    - Replace the `epoch_deadline` with the desired duration in seconds.
+    - Replace the `database` with the directory location of the persistent storage. However, this value is currently unused.
```

Comment:
It's not actually unused though, because the database gets created.

Did we have ticket about temporarily removing the storage stuff before the release?  Is that something we still want to do?